Create 05-review file and addition of possible topics (#23)
empa-scientific-it/how-to-git: src/content/docs/guides/05-review.mdx

Reviews let teammates give feedback, helping you see your work from a fresh perspective and improve it.
Reviews are not just for code; they can also cover documentation, metadata files, or settings.
They can even be set as required, to keep quality and consistency high.
This way, no changes can be merged before review and approval.
On GitHub, these reviews are commonly known as pull request (PR) reviews.

<Aside type="tip">
Be friendly and specific while reviewing: Tell what needs to be changed and how, and give a reason if it is not clear.
</Aside>

### Request a review as an author
A review should be requested, if the work on a topic has finished.
Finishing the work on a topic means you wish to merge your branch to the main branch.
To do this, you open a Pull Request (PR) and choose someone specific to review it.
It is often best to choose a reviewer who was not directly involved in the development of this topic but is familiar with it.

- First, open a [Pull Request](/guides/parallel-topics/#merge-branches).
- On the top right you can choose who to assign as a reviewer.

> - Now you need to wait for the reviewer to finish the review.

Reviewer: Can you provide a picture here with an example?

### Make changes as a reviewer
As a reviewer you will receive a notification (depending on your settings) that you are requested to review a PR.

> 1. When reviewing, you will see a splitscreen view where on the left, highlighted with red, are the previous lines of text that have been edited, and on the right, highlighted with green, are the newly added changes

Reviewer: ```suggestion
1. When reviewing, you will see a splitscreen view where on the left, highlighted in red, are the original lines of text that have been edited, and on the right, highlighted in green, are the newly added changes
```

2. Go to a line of text that you would review, click on the plus symbol :heavy_plus_sign:
    ![Add review comment](src/assets/add-review-comment.png)
3. Either add a comment, or choose to make a suggestion (click on the plusminus symbol), which gives you the option to propose changes to one or more lines
    - A suggestion can simply be accepted by the author, whereas a comment just gives a hint for a change
4. Click the green button to start the review or to add more comments to an ongoing review
5. When you are finished with your comments click the green button at the top to finish your review
    - This will send a notification to the author for further actions

>     <CustomVideoPlayer path="/src/assets/Recording-submit-review.mp4" />

Reviewer: This video is not working on the deployed version
